aspnetcore/blazor/components/index.md

```diff
@@ -1865,10 +1865,6 @@ For more information, see the following resources:
 * <xref:mvc/views/tag-helpers/builtin-th/component-tag-helper>
 * <xref:blazor/components/integration>
 
-## `IHttpContextAccessor`/`HttpContext`
-
-[!INCLUDE[](~/blazor/security/includes/httpcontext.md)]
-
```

Comment:
Should this include be replaced with a link to the new article?

Reply:
It's going to be very visible in the ToC, so I'd say 'no,' but it's up to you if you prefer the section remain.